src/overflow/explainer.md

```diff
@@ -284,6 +284,76 @@ Below, we'll explore some of the features
 that are still missing or difficult
 for authors to get right.
 
+### Carousel example
+
+![Carousel boxes](../carousel-boxes.svg)
+
+As a strawman example,
+a developer should be able to use the following HTML
+to create the visualized carousel:
+
+```html
+<carousel>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+</carousel>
+```
+
+The visualized carousel
+can be constructed
+with the following style
+dynamically generating all of the required components:
+
+```css
+carousel {
+  grid-template-columns: [previous] auto [scroller] 1fr [next] auto;
+  grid-template-rows: 
+    [carousel] 1fr 
+    [markers] auto;
```

Comment:
Not sure I understand this grid template. It creates a few named lines, but no named areas. Should it be something like this (to match the diagram)?

```suggestion
  grid-template: 
    'previous scroller next' 1fr
    '. markers .' auto
    / auto 1fr auto;
```